packages/shared/styles/helpers/_colors.scss

```diff
@@ -0,0 +1,21 @@
+@each $color, $value in $colors {
+  // Atomic utility classes for text and bg colors
+  .text-#{$color} {
+    color: $value !important;
+  }
+  .bg-#{$color} {
+    background-color: $value !important;
+  }
+
+  // General color modifiers
+  // May be overwritten by specific component modifiers
+  .is-#{$color} {
+    color: color-yiq($value) !important;
+    background-color: $value !important;
+    border-color: darken($value, 5) !important;
```

Comment:
No `!important` please :(

Reply:
haha I don't like it too, but removing it the styles will not be applied, unless we change the SCSS import order, loading the helpers after components styles

Reply:
As the components may already have they own default colors, these color modifiers should **replace styles**

Reply:
I don't think we need `important` though? users should make sure not to overwrite these styles by mistake right?

Reply:
If we want to make these color modifier to overwrite then we should use `scope` or css module on component itself instead of using important.

Reply:
@mayashavin it's not about the user, it's about the component default styles.

Take a look o `SfButton` as example, it has primary color bg by default, we have to overwrite it when using `color-warning` modifier, right? I'm not sure but I think using `scope` or CSS module would not solve it.

The only solution I'm seeing to skip using `!important` is loading the helpers after components styles.

Reply:
I agree with @leomp12 here